packages/phone-number-privacy/signer/src/server.ts

```diff
@@ -98,7 +99,8 @@ export function startSigner(
       new PnpQuotaIO(
         config.api.phoneNumberPrivacy.enabled,
         config.api.phoneNumberPrivacy.shouldFailOpen, // TODO (https://github.com/celo-org/celo-monorepo/issues/9862) consider refactoring config to make the code cleaner
-        kit
+        kit,
+        Number(config.blockchain.timeout_ms ?? FULL_NODE_TIMEOUT_IN_MS)
```

Comment:
I think you can just pass config.blockchain.timeout_ms here since the default is enforced elsewhere